src/codeu/chat/server/ViewDatabase.java

```diff
@@ -0,0 +1,371 @@
+package codeu.chat.server;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+import codeu.chat.common.*;
+import codeu.chat.util.Time;
+import codeu.chat.util.Logger;
+import codeu.chat.util.Uuid;
+import codeu.chat.util.store.StoreAccessor;
+
+/**
+ * Created by strobe on 4/04/17.
+ */
+public final class ViewDatabase {
+
+    private final SQLFormatter sqlFormatter = new SQLFormatter();
+
+    public static Collection<User> getUsers(Collection<Uuid> ids){
+        final Collection<User> found = new HashSet<>();
+
+        System.out.println("Accessing ViewDatabase");
+
+        Connection connection = null;
+        Statement stmt = null;
+
+        try {
+            Class.forName("org.sqlite.JDBC");
+            connection = DriverManager.getConnection("jdbc:sqlite:./bin/codeu/chat/codeU_db/ChatDatabase.db");
+            connection.setAutoCommit(false);
+
+            for(final Uuid id : ids) {
+                stmt = connection.createStatement();
+                ResultSet rs = stmt.executeQuery( "SELECT * " +
+                        "FROM USERS " +
+                        "WHERE  ID = "+SQLFormatter.sqlID(id)+";" );
```

Comment:
I think you could get rid of the for loop if you used OR in the query.  (i.e. ID="a" OR ID = "b"). If you did this then I think you might be able to create a helper function a that takes a table name and a set of IDs and simply return a ResultSet you could iterate over. I think that could result in the removal of a lot of duplicated code in this file.